src/test/scala/io/iohk/ethereum/ledger/LedgerSpec.scala

```diff
@@ -663,10 +663,35 @@ class LedgerSpec extends FlatSpec with PropertyChecks with Matchers {
     }
   }
 
+  it should "create sender account if it does not exists" in new TestSetup {
+
+    val inputData = ByteString("the payload")
+
+    val newAccountKeyPair: AsymmetricCipherKeyPair = generateKeyPair()
+    val newAccountAddress = Address(kec256(newAccountKeyPair.getPublic.asInstanceOf[ECPublicKeyParameters].getQ.getEncoded(false).tail))
+
+    val mockVM = new MockVM((pc: Ledger.PC) => {
+      pc.env.inputData shouldEqual ByteString.empty
+      createResult(pc, defaultGasLimit, defaultGasLimit, 0, None, returnData = ByteString("contract code"))
+    })
+    val ledger = new LedgerImpl(mockVM, blockchainConfig)
+
+    val tx: Transaction = defaultTx.copy(gasPrice = 0, receivingAddress = None, payload = inputData)
```

Comment:
Do you think it's worthy to write a test where the execution fails because `gasPrice` is not zero?